src/ray/core_worker/test/actor_task_submitter_test.cc

```diff
@@ -654,8 +635,28 @@ TEST_P(ActorTaskSubmitterTest, TestActorRestartFailInflightTasks) {
   const auto death_cause = CreateMockDeathCause();
   submitter_.DisconnectActor(
       actor_id, 1, /*dead=*/false, death_cause, /*is_restartable=*/true);
+  ASSERT_EQ(submitter_.NumInflightTasks(actor_id), 0);
+  // RPC callbacks are delayed.
+  ASSERT_EQ(worker_client_->callbacks.size(), 2);
 
-  // The task replies are now received. Since the tasks are already failed, they will not
+  // Task2 and task3 are retried.
+  task2.GetMutableMessage().set_attempt_number(task2.AttemptNumber() + 1);
+  task2.GetMutableMessage().mutable_actor_task_spec()->set_actor_counter(3);
```

Comment:
what is the "actor_counter"? is it the number of restarts?

Reply:
if so we should rename that...

Reply:
```
  // Number of tasks that have been submitted to this actor so far.
  uint64 actor_counter = 5;
```

We often call it seq_no. I'll create a rename PR right after this one.